README.md

```diff
@@ -1 +1,32 @@
-Feedhenry Sample Service Using MongoDB Data for Datasources
+# Feedhenry Sample Service Using MongoDB Data for Datasources
+
+This is a sample RESTful service to load a list of **Assets** stored in a MongoDB database that is properly formatted as a Data Source which can be used in Apps created with Form Builder.
```

Comment:
"properly formatted for a Data Source" I think is more correct.

I don't think you need the "which can be used in Apps created with Form Builder." part.